packages/node-cli/src/services/TranslationKeys.js

```diff
@@ -140,11 +140,14 @@ module.exports = {
 			IMPORTING_FILES: 'COMMAND_IMPORTFILES_MESSAGES_IMPORTING_FILES',
 			LOADING_FILES: 'COMMAND_IMPORTFILES_MESSAGES_LOADING_FILES',
 			LOADING_FOLDERS: 'COMMAND_IMPORTFILES_MESSAGES_LOADING_FOLDERS',
+			UPDATING_FILE: 'COMMAND_IMPORTFILES_MESSAGES_UPDATING_FILE',
```

Comment:
Following the previous comment from `ImportFilesAction.js`  this message probably also needs to be deleted.